README.md

```diff
@@ -1,3 +1,4 @@
+![Tests](https://github.com/pglass/sqlitis/workflows/Test/badge.svg)
```

Comment:
This may show whatever the latest build is for the workflow, rather than the latest build for the master branch...

Reply:
https://docs.github.com/en/free-pro-team@latest/actions/managing-workflow-runs/adding-a-workflow-status-badge#using-the-branch-parameter